hack/verify-gofmt.sh

```diff
@@ -52,3 +52,12 @@ if [[ -n "${bad_files}" ]]; then
   echo "${bad_files}"
   exit 1
 fi
+
+# Find comments not prefaced with a space.  gofmt doesn't do this.
+bad_files=$(find_files | xargs grep '//[^ ]' | grep -v "://")
+if [[ ${bad_files} > 0 ]]; then
```

Comment:
This test just doesn't make sense to me, I assume you wanted `if [[ -n "${bad_files}" ]]`  but maybe a bash array is really what you want...

```
bad_files=($(find_files | xargs grep '//[^ ]' | grep -v "://"))
if [[ ${#bad_files[@] -ne 0 ]]; then
  echo "Comments in the above files need to be in idiomatic format i.e. '// hello kube'"
  echo "We found ${#bad_files[@]} files with unidiomatic comments" 
  echo "${bad_files[@:0:4]}"
fi
```

Untested, but I think that last 'echo' will be correct....